### PR TITLE
Casmnet 1258

### PR DESCRIPTION
**Pull request by kosonenj**
### Summary and Scope

Description: 

Fix for: 

[1.2.5-develop]

    Add NMN network for 1.0 to ssh allowed into switches because of BGP DOCS in 1.0 allowing it.
    Remove router ospfv3 from 1.0/1.2.
    Fixed ACL, OSPF, BGP config files
    Fixed test templates for ACL, OSPF, BGP

PR checklist (you may replace this section):
- [x] I have run `nox` locally and all tests, linting, and code coverage pass.
- [x] I have added new tests to cover the new code
- [x] My code follows the style guidelines of this project
- [ ] If adding a new file, I have updated pyinstaller.py
- [x] I have updated the appropriate Changelog entries in readme.md
- [x] I have incremented the version in the readme.md
- [x] I have incremented the version in `canu/.version`

### Issues and Related PRs

* Resolves `CASMNET-1258`
* Relates to `Issue`
* Requires `Issue`

### Testing

Tested on:  local NOX

* List of virtual system tested.
* List of physical systems tested.

### CODE REVIEW

**Comment by kosonenj** [comment]
Need review

File: network_modeling/configs/templates/1.0/aruba/common/acl.j2 [hunk 1]
```diff
@@ -2,6 +2,7 @@
 access-list ip mgmt
 {% set sequence = 10 %}    {{ sequence }} comment ALLOW SSH, HTTPS, AND SNMP ON HMN SUBNET
 {% set sequence = sequence+10 %}    {{ sequence }} permit tcp {{ variables.HMN_NETWORK_IP }}/{{ variables.HMN_NETMASK }} any eq ssh
+{% set sequence = sequence+10 %}    {{ sequence }} permit tcp {{ variables.NMN_NETWORK_IP }}/{{ variables.NMN_NETMASK }} any eq ssh
```
Comment (lukebates123) [inline network_modeling/configs/templates/1.0/aruba/common/acl.j2:5]: This should be fine for 1.0

**Review by lukebates123 — APPROVED** [review]

File: canu/generate/switch/config/config.py [hunk 2]
```diff
@@ -733,6 +733,9 @@ def generate_switch_config(
             override_config_hier = HConfig(host=host)
             override_config_hier.load_from_string(switch_config)
             override_config_hier.add_tags(override[switch_name])
+            # add ! to the end of the aruba banner.
+            banner = override_config_hier.get_child("contains", "banner")
```
Comment (trad511) [inline canu/generate/switch/config/config.py:737]: This makes me sad.

File: docs/nox_testing.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Nox testing
```
Comment (trad511) [inline docs/nox_testing.md:1]: Nice.

**Review by trad511 — APPROVED** [review]

**Pull request merged**